test/e2e/ha/openstack_services_test.go

```diff
@@ -96,6 +115,15 @@ func TestHAOpenStackServices(t *testing.T) {
 			t.Run("then all services should have 3 ready replicas", func(t *testing.T) {
 				assertOpenStackReplicasReady(t, w, 3)
 			})
+
+			t.Run("then the keystone service should handle request for a token", func(t *testing.T) {
+				t.Skip()
+				keystoneProxy := proxy.NewSecureClient("contrail", "keystone-keystone-statefulset-0", 5555)
```

Comment:
As we are introducing services. I think we should try to proxy traffic not to a pod but to service.

I think this might be it https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/

Reply:
will we be able to access ClusterIP this way?

Reply:
We will use service name here

Reply:
We have to change the whole path like here https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#manually-constructing-apiserver-proxy-urls